### PR TITLE
Add initial app/schema for stormwater features

### PR DESCRIPTION
**Pull request by steventlamb**

### CODE REVIEW

**Comment by RickMohr** [comment]
Looks like there are a few broken tests.

File: opentreemap/treemap/audit.py [hunk 1]
```diff
@@ -140,13 +140,13 @@ def approve_or_reject_audits_and_apply(audits, user, approved):
...
-    from treemap.models import MapFeature
+    from treemap.models import MapFeaturePhoto
     if roles is None:
         roles = Role.objects.filter(instance=instance)
     if models is None:
-        # MapFeature is "Authorizable", but it is effectively abstract
-        # Only it's subclasses should have permissions added
-        models = all_subclasses(Authorizable) - {MapFeature, PendingAuditable}
+        # MapFeaturePhoto is an exception to the usual pattern - it is
+        # the only class that we use directly *and* inherit.
```
Comment (RickMohr) [inline opentreemap/treemap/audit.py]: Much better. How about this rewritten comment:

We need permissions only on those subclasses of `Authorizable` which we instantiate. Those are the leaf nodes of the subclass tree, plus `MapFeaturePhoto` (which has subclass `TreePhoto`).

**Comment by steventlamb** [comment]
@RickMohr updated

**Comment by RickMohr** [comment]
+1 thanks.

**Pull request merged**